pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
+		span, ctx = ot.StartSpanFromContext(ctx, fmt.Sprintf("%s-%d", methodName, i))
+		span.LogFields(log.Object("gRPC request", request))
```

Comment:
I can remove logging req/resp in this PR and we can add it back when we get around to implementing tracing options in general.